src/planscape/plan/models.py

```diff
@@ -47,46 +47,32 @@ class Project(models.Model):
     plan = models.ForeignKey(Plan, on_delete=models.CASCADE)  # type: ignore
 
     # Project Parameters:
+ 
+    # TODO: Limit number of allowed priorities
+    priorities = models.ManyToManyField('conditions.Condition')
```

Comment:
I don't think this works: this will only allow one priority per project.

I think ArrayField is better: https://docs.djangoproject.com/en/4.1/ref/contrib/postgres/fields/#arrayfield

Reply:
I'm reading something different in [way-to-use-a-manytomanyfield-in-django](https://www.sankalpjonna.com/learn-django/the-right-way-to-use-a-manytomanyfield-in-django). I considered ArrayField, but there are some limitations as well - I think it acts essentially like a list of strings.

Reply:
Ah.  I wrote it out, looked at the document you linked.  Now I understand that it essentially creates a table

ProjectPriority:
  project = models.ForeignKey(Project)
  priority = models.ForeignKey(Condition)

under the hood.  That works.

Reply:
I'll add this to the PR description to make it clearer